package.json

```diff
@@ -40,7 +40,9 @@
     "fsevents"
   ],
   "dependencies": {
-    "@fortawesome/fontawesome-free-webfonts": "^1.0.9",
+    "@fortawesome/fontawesome": "1.1.8",
+    "@fortawesome/fontawesome-free-solid": "5.0.13",
```

Comment:
Why are these needed (what requires these)? We've previously moved off of them due to performance issues with every `render()` re-creating inline SVGs

Reply:
I dunno, it gave me errors back when I started developing on the branch. I think I can get rid of them.
Should I just keep `fontawesome-free-webfonts` or delete that as well?

Reply:
> Should I just keep fontawesome-free-webfonts or delete that as well?

Keep keep, that's used by the file-picker ;)